common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -464,6 +525,21 @@ innovation_troll_tactics = {
 	flag = global_maa
 }
 
+innovation_tuskarr_tactics = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @trapper
+
+	potential = {
+		this.culture_group = culture_group:tuskarr_group
+	}
+
+	unlock_maa = trapper
+	unlock_maa = turtle_rider
```

Comment:
Such debuffs and buffs are too small and barely noticeable. They're noticeable starting from 10% of the base value.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/151370839-1931ea16-cb1a-4d68-aab3-67fce5904cd6.png)

</details>